client/my-sites/plans/jetpack-plans/product-store/items-list/bundles-list.tsx

```diff
@@ -7,11 +7,10 @@ import { MostPopular } from './most-popular';
 import type { BundlesListProps } from '../types';
 
 export const BundlesList: React.FC< BundlesListProps > = ( {
-	createCheckoutURL,
-	duration,
-	onClickPurchase,
+	storeItemInfo,
```

Comment:
Is this needed? I assume this is something you forgot to remove while refaactoring

Reply:
Yeah it is also causing the unit test error.

Reply:
Thanks for spotting this!